src/tiled/gotodialog.h

```diff
@@ -0,0 +1,42 @@
+#ifndef GOTODIALOG_H
```

Comment:
Of course, this file should get the license header as well.

Also, please use `#pragma once` instead of header guards (I've replaced them all a while ago: 1ecd50adb793dddc35ca3e161e396dc95a89f1a7).